include/oneapi/dpl/pstl/algorithm_impl.h

```diff
@@ -3021,23 +3020,22 @@ __pattern_merge(_Tag, _ExecutionPolicy&&, _ForwardIterator1 __first1, _ForwardIt
 }
 
 template <typename _Tag, typename _ExecutionPolicy, typename _It1, typename _Index1, typename _It2, typename _Index2,
-          typename _OutIt, typename _Index3, typename _Comp>
+          typename _OutIt, typename _Index3, typename _Comp, typename _Proj1, typename _Proj2>
 std::pair<_It1, _It2>
 ___merge_path_out_lim(_Tag, _ExecutionPolicy&&, _It1 __it_1, _Index1 __n_1, _It2 __it_2, _Index2 __n_2,
-                      _OutIt __it_out, _Index3 __n_out, _Comp __comp)
+                      _OutIt __it_out, _Index3 __n_out, _Comp __comp, _Proj1 __proj1, _Proj2 __proj2)
 {
```

Comment:
```suggestion
{
    static_assert(__is_serial_tag_v<_Tag> || __is_parallel_forward_tag_v<_Tag>);
```

Reply:
Let's abstain from making changes unrelated to the goal of the PR.

Reply:
Could you create sone new issue and add these `static_assert` in the separate PR ?

Reply:
A separate PR: https://github.com/uxlfoundation/oneDPL/pull/2324

Reply:
Thanks!